### PR TITLE
Boost part 3

### PR DESCRIPTION
**Pull request by Cast0001**

### CODE REVIEW

**Review by dfkadyr — APPROVED** [review]
🦅

File: src/methods/vault/requests/getBoost.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  validateArgs.address({ vaultAddress, userAddress })
+
+  const boost: Output = {
+    shares: 0n,
+    percent: 0,
+    maxMintShares: 0n,
+    isProfitable: false,
+  }
+
+  if (options.network === Network.Mainnet) {
```
Comment (mike-diamond) [inline src/methods/vault/requests/getBoost.ts:32]: Should we allow holesky as well?

File: src/methods/vault/requests/getBoost.ts [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    const maxMintAssets = stakedAssets * ltvPercent / constants.blockchain.amount1
+    const maxMintShares = await contracts.base.mintTokenController.convertToShares(maxMintAssets)
+
+    const boostPercent = maxMintShares ? Number(
+      new BigDecimal(boostShares)
+        .multiply(100)
+        .divide(maxMintShares)
+        .decimals(2)
+        .toString()
```
Comment (mike-diamond) [inline src/methods/vault/requests/getBoost.ts:61]: It would be helpful to add `toNumber()` method

**Review by mike-diamond — COMMENTED** [review]
👍

**Pull request merged**